ocs_ci/deployment/deployment.py

```diff
@@ -2816,6 +2822,31 @@ def deploy(self):
             rbddops.deploy()
         self.deploy_dr_policy()
 
+        # Enable cluster backup on both ACMs
+        for i in acm_indexes:
+            config.switch_ctx(i)
+            enable_cluster_backup()
+        # Configuring s3 bucket
+        self.meta_obj.get_meta_access_secret_keys()
+        # bucket name formed like '{acm_active_cluster}-{acm_passive_cluster}'
+        self.meta_obj.bucket_name = build_bucket_name(acm_indexes)
+        # create s3 bucket
+        create_s3_bucket(
+            self.meta_obj.access_key,
+            self.meta_obj.secret_key,
+            self.meta_obj.bucket_name,
+        )
+        create_generic_credentials(
+            self.meta_obj.access_key, self.meta_obj.access_key, acm_indexes
+        )
+        # Reconfigure OADP on all ACM clusters
+        for i in acm_indexes:
+            config.switch_ctx(i)
+            create_dpa(self.meta_obj.bucket_name)
+        # Only on the active hub enable managedserviceaccount-preview
+        config.switch_acm_ctx()
+        enable_managed_serviceaccount()
+
 
 class MDRMultiClusterDROperatorsDeploy(MultiClusterDROperatorsDeploy):
```

Comment:
Can we keep this class name generic as DR?

Reply:
This class belongs to MDR So we cannot rename it